packages/api-fetch/src/index.js

```diff
@@ -80,48 +77,9 @@ const defaultFetchHandler = ( nextOptions ) => {
 		}
 	);
 
-	const parseResponse = ( response ) => {
```

Comment:
The parsing and error normalizing here was extracted to be reused in middlewares.